lorien/UI/Components/KeyBindingsLineBindings.gd

```diff
@@ -0,0 +1,43 @@
+extends HBoxContainer
+
+# -------------------------------------------------------------------------------------------------
+signal modified_binding(bindings_data)
+
+# -------------------------------------------------------------------------------------------------
+var bindings_data := {}
```

Comment:
Is the local `bindings_data` variable accessed from outside? If not it should be prefixed with an underscore to mark it as private.

Reply:
Underscore it is!